src/graphql/resolvers/GDPRUser.js

```diff
@@ -1,10 +1,11 @@
-import { createUser, getUser } from '../../dataLayer/mongo/user';
+import { createUser, getUser, deleteUser } from '../../dataLayer/mongo/user';
 
 export const userResolvers = {
   Query: {
     getGDPRUser: getUser
   },
   Mutation: {
-    createUser
+    createUser,
+    deleteGDPRUser: deleteUser
```

Comment:
I think this can just be `deleteUser` instead.

Reply:
Then I will have to change the schema too, which was not defined by me. Is that all right?

Reply:
Sure!